Sources/Commands/SwiftTestTool.swift

```diff
@@ -288,6 +287,56 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                 executionStatus = .failure
             }
         }
+
+        if options.shouldEnableCodeCoverage {
+            let buildParameters = try self.buildParameters()
+            let fs = Basic.localFileSystem
+            mergeRawDataFiles(fs: fs, path: buildParameters.buildPath.appending(component: "codecov"))
+            if buildParameters.triple.isDarwin() {
+                let graph = try loadPackageGraph()
+                let component = graph.rootPackages[0].name + "PackageTests"
```

Comment:
Is this right? I think we should iterate over `rootPackages` and generate a report for each. In practice, it is going to be always one right now, but we shouldn't hardcode that assumption here.